src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
+        Ball ball = new Ball();
```

Comment:
You can directly return the new Ball object without storing it in a temporary variable. This will make the code more concise. For example: 'return new Ball(color, number);' if the Ball class has an appropriate constructor.